src/Components/BarAndLineChartsWrapper/renderLineInBarChart/renderDataPoints.tsx

```diff
@@ -23,6 +23,7 @@ export const renderDataPoints = props => {
     const customDataPoint = item.customDataPoint || lineConfig.customDataPoint;
     const value =
       item.value ?? item.stacks.reduce((total, item) => total + item.value, 0);
+    if (item.hideDataPoint) return null;
```

Comment:
Please put this statement at the top in line 19
Add the condition- ` || item.hideDataPoint` in the same if statement. (for early return)

Reply:
@Abhinandan-Kushwaha modified to have early return